src/unix/fts.cc

```diff
@@ -1,5 +1,9 @@
 #include <string>
-#define __THROW // weird error on linux
+
+// weird error on linux
+#undef __THROW
```

Comment:
Adding this fixes a warning that pops up on build

Reply:
To be sure that this doesn't cause a compile error on some systems:
```c
#ifdef __THROW
#undef __THROW
#endif
#define __THROW
```

Reply:
Thanks for this :)